frontend/src/app/components/calculate-size/calculate-size.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import {Router} from '@angular/router';
+import {PostDimensionsService} from '../../services/post-dimensions.service';
+import {DummyParcelDimensions} from '../../common/dummy-parcel-dimensions';
+import {NGXLogger} from 'ngx-logger';
+
+@Component({
+  selector: 'app-calculate-size',
+  templateUrl: './calculate-size.component.html',
+  styleUrls: ['./calculate-size.component.css']
+})
+export class CalculateSizeComponent implements OnInit {
+  size: string;
+  constructor(private router: Router, private postDimensionsService: PostDimensionsService, private logger: NGXLogger) {
+  }
+
+  ngOnInit(): void {
+  }
+
+  calcsml(length: number, width: number, height: number): void {
+    const dummyParcelDimensions = new DummyParcelDimensions();
```

Comment:
Please add constructor in DummyParcelDimensions() to accept and set these 3 fields

Reply:
done